### DISCUSSION TITLE
Document that CSM disbled blocks updates

### DISCUSSION THREAD

**Discussion by kleinph** [fwupd/fwupd#3525] [General]
**Is your feature request related to a problem? Please describe.**
I had trouble to update _System Firmware_ on my ThinkPad T480s and got a message that the update failed to run after reboot with a link to the according [wiki page](https://github.com/fwupd/fwupd/wiki/LVFS-Triaged-Issue:-Failed-to-run-update-on-reboot).

But _Boot Order Lock_ as well as _Secure Boot_ were disabled. After enabling _CSM Support_ the update installed successfully.

**Describe the solution you'd like**
Document on the [Failed to run update on reboot wiki page](https://github.com/fwupd/fwupd/wiki/LVFS-Triaged-Issue:-Failed-to-run-update-on-reboot) that _CSM Support_ should be enabled.

**Describe alternatives you've considered**
If _CSM Support_ should have no effect on firmware updates this have to be investigated as a bug.

**Additional context**
This may also be related to the fact that _UEFI only_ is enabled.
Sidenote: turning on CSM support doesn't fix may problems with installing Intel ME updates :(

**Comment by superm1** [comment]
I don't expect CSM support to modify this behavior, particularly because having secure boot enabled will implicitly have CSM disabled.  This sounds like it's in BIOS bug territory.  On the next update that occurs, can you please capture more details with CSM disabled?

**Reply by kleinph** [reply]
Sorry, I don't understand the reason for closing: the wiki page states to turn off secure boot.

How do I capture more details?
Thanks

**Reply by superm1** [reply]
I didn't close it, I converted it to a discussion.
The reason for turning off secure boot is as a debugging tactic only.  There was a recent failure for shim chainloading binaries for example, and turning off secure boot mitigates this bug.

When the issue happens, follow the UEFI bug template.  It will advise what stuff to capture so we can work through where the issue is.

**Reply by kleinph** [reply]
> I didn't close it, I converted it to a discussion.

Sorry for being unprecise. I understand.

> When the issue happens, follow the UEFI bug template. It will advise what stuff to capture so we can work through where the issue is.

Tanks, will do that if it occurs again.
